### PR TITLE
Sources update from the main repo

### PR DESCRIPTION
**Pull request by vadym-vorotilin**
These changes fail two tests. I'm not sure if they should be merged in this case.

image

### CODE REVIEW

File: src/license.txt [hunk 1]
```diff
@@ -1,25 +0,0 @@
-MIT License
```
Comment (mavimaec) [inline src/license.txt:1]: I think this license file got mistakenly removed in this PR

**Review by mavimaec — CHANGES_REQUESTED** [review]
Thanks for the merge @Vadim-Vorotilin ! I have a change request below for you. Also, do you think you can investigate the unit test failure?

File: src/license.txt (hunk 1 above)
Comment (vadym-vorotilin) [reply src/license.txt:1]: @mavimaec  The license file is in the root folder. I think this one was a duplicate.

**Review by mavimaec — APPROVED** [review]
👍 good eye on the license.txt!

**Review by cdiggins — APPROVED** [review]
Thank you!

**Pull request merged**